sklearn/cluster/tests/test_kernel_kmeans.py

```diff
@@ -0,0 +1,56 @@
+""" Testing for Kernel K means"""
+
+import numpy as np
+from scipy import sparse as sp
+
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import assert_raises
+
+from sklearn.cluster import KernelKMeans
+from sklearn.datasets.samples_generator import make_blobs
+from sklearn.metrics.cluster import v_measure_score
+
+centers = np.array([
+    [0.0, 5.0, 0.0, 0.0, 0.0],
+    [1.0, 1.0, 4.0, 0.0, 0.0],
+    [1.0, 0.0, 0.0, 5.0, 1.0],
+])
+n_samples = 100
+n_clusters, n_features = centers.shape
+X, true_labels = make_blobs(n_samples=n_samples, centers=centers,
+                            cluster_std=1., random_state=42)
+X_csr = sp.csr_matrix(X)
+
+
+def test_kernel_kmeans_dtype():
+    rnd = np.random.RandomState(0)
+    X = rnd.normal(size=(40, 2))
+    X = (X * 10).astype(np.uint8)
+    clf = KernelKMeans().fit(X)
+    pred_x = clf.predict(X)
+    assert_array_equal(clf.labels_, pred_x)
+
+
+def _check_fitted_model(clf):
```

Comment:
I was using this for running kernel tests and forgot to remove it when i was making some changes. I have added kernel tests again.